src/ripple/rpc/impl/Handler.cpp

```diff
@@ -166,8 +166,13 @@ class HandlerTable
     template <std::size_t N>
     explicit HandlerTable(const Handler (&entries)[N])
```

Comment:
Right, the current code allow different handlers for different versions, in addition to one handler dealing with different API versions internally. I think it is more flexible in case people want different handlers. But I don't against simplifying it. Love to hear what @cjcobb23 has to say.

Reply:
I prefer that the handlers distinguish between different API versions internally. It's easier to reuse common logic within the handlers this way, especially when the version differences are small. For drastically different handlers, I think we would just do something similar to how some of the reporting mode handlers were implemented: https://github.com/ripple/rippled/blob/9d89d4c1885e13e3c0aa7cdb4a389c6fbfd3790a/src/ripple/rpc/handlers/Tx.cpp#L173

Reply:
However, if the handlers themselves distinguish between different versions internally, I don't think there is a way for different versions of the handler to have different conditions. The condition check happens before actually calling the handler. I don't know how likely it is that different versions of the handlers have different conditions, but it's probably possible.

Reply:
@pwang200 @scottschurr I think if we want to support the case where different versions of the handler have different conditions, we need to keep this table. However, maybe we should remove the table for now and add it back if/when we need different conditions for different versions of the handler. Thoughts?

Reply:
Speaking only for myself, I'd rather remove the table now.  We don't know whether we'll need this particular flexibility (or something else) in the future.  I'd prefer to postpone the complexity until it's needed.

Reply:
Thanks for the votes! Let's remove it.

Reply:
Changed.